templates/wagtail/files/myapp/wsgi.py

```diff
@@ -0,0 +1,16 @@
+"""
+WSGI config for myapp project.
+
+It exposes the WSGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.1/howto/deployment/wsgi/
+"""
+
+import os
+
+from django.core.wsgi import get_wsgi_application
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'myapp.settings')
```

Comment:
Is "myapp" a Wagtail thing, or something we're doing?

Reply:
It's taken from the Django 2 template naming, which does seem to be a Django naming convention at least for examples.

Reply:
Hm.  Not a fan, but we have nothing better I suppose.